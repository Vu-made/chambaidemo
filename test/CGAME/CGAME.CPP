#include <bits/stdc++.h>
#define N 100005
using namespace std;
int a[N];
int s1 = 0, s2 = 0, n;
int main()
{
    freopen("CGAME.INP","r",stdin);
    freopen("CGAME.OUT","w",stdout);
    cin >> n;
    for (int i=1; i<=n; i++) cin >> a[i];
    int i = 1, j = n, d = 0;
    while (i <= j)
    {
        if (d == 0) s1 += max(a[i],a[j]);
        else s2 += max(a[i],a[j]);

        if (a[i] > a[j]) i++;
        else j--;
        d = 1 - d;
    }
    cout << s1 << " " << s2;
    return 0;
}
