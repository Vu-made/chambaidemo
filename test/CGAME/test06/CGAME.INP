88
85 60 36 6 10 9 63 88 59 40 58 65 12 50 8 46 35 82 83 28 80 16 42 57 2 84 81 87 48 27 21 51 33 61 4 62 30 24 22 71 15 34 86 72 75 53 37 19 74 18 45 70 49 31 41 77 5 23 3 76 68 47 52 73 25 64 39 38 11 7 79 17 1 26 67 55 56 54 20 66 44 13 43 29 69 14 78 32 