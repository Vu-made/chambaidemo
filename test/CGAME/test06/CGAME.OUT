2171 1745